### PR TITLE
#144 throws should be real

### PR DESCRIPTION
**Pull request by krzyk**
#144 
* using fb-contrib rule `BED_BOGUS_EXCEPTION_DECLARATION` to check for throws declaration with reality
* one drawback is that this check ignores public methods - this is sometimes required for APIs (see mebigfatguy/fb-contrib#92), but the good news is that it works for final methods/classes so it should cover almost all classes (as qulice enforces them to be final if they are not abstract)

### CODE REVIEW

**Comment by davvd** [comment]
@krzyk Thanks, I will find someone to review this PR soon

**Comment by davvd** [comment]
@dmzaytsev this pull request is for you,please review

File: qulice-findbugs/src/test/java/com/qulice/findbugs/FindBugsValidatorTest.java [hunk 1]
```diff
@@ -54,6 +55,30 @@ public void passesCorrectFilesWithNoExceptions() throws Exception {
     }
 
     /**
+     * FindbugsValidator can report incorrectly added throws.
+     * @throws Exception If something wrong happens inside
+     */
+    @Test(expected = ValidationException.class)
+    public void reportsIncorrectlyAddedThrows() throws Exception {
```
Comment (dmzaytsev) [inline qulice-findbugs/src/test/java/com/qulice/findbugs/FindBugsValidatorTest.java]: @krzyk can we add one more test ?
to make sure FindbugsValidator doesn't report correctly added throws

**Comment by dmzaytsev** [comment]
@krzyk please see my comment

**Comment by krzyk** [comment]
@dmzaytsev updated

**Comment by dmzaytsev** [comment]
@krzyk thanks

**Comment by dmzaytsev** [comment]
@rultor merge

**Comment by dmzaytsev** [comment]
@rultor deploy

**Comment by davvd** [comment]
@dmzaytsev Much appreciated! **20 mins** was added to your account, payment ID `76665344`, time spent is 24 hours and 45 mins... extra minutes for review comments (c=5)... +20 to your rating, your total score is [+3063](http://www.netbout.com/b/35166?open=rating)

**Comment by davvd** [comment]
@rultor please deploy

**Pull request merged**